app/javascript/components/home/HomePage.jsx

```diff
@@ -27,8 +33,16 @@ export default function HomePage() {
     [currentUser.signed_in],
   );
 
+  useEffect(() => {
+    document.cookie = `token=${inviteToken};path=/;`;
```

Comment:
I suspect that this will always create the cookie,

Reply:
Why we don't host this logic on signup where it will check the validity of the request,
It will further set any required cookies and query params for the signup request,

Reply:
So the invitation URL is a `.../signup?inviteToken=xxx`